crypto/fipsmodule/cpucap/cpu_aarch64_apple.c

```diff
@@ -71,6 +90,14 @@ void OPENSSL_cpuid_setup(void) {
     OPENSSL_armcap_P |= ARMV8_SHA512;
   }
 
+  if (has_hw_feature("hw.optional.armv8_2_sha3")) {
+    OPENSSL_armcap_P |= ARMV8_SHA3;
+  }
+
+  if (is_brand("Apple M1")) {
```

Comment:
We must remember to add additional string matching here when new laptops are released :)